gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/resources/webroot/views/mfa_enroll.html

```diff
@@ -290,7 +290,6 @@
         document.getElementById('mfa-enroll-step2').classList.add('hide');
         document.getElementById('call-factor').classList.remove('hide')
         document.getElementById('sms-factor').classList.remove('hide')
```

Comment:
I was also concern about those two but I tested this and it seems to be working fine. This removal is because sms and call factor uses the same template. 

https://github.com/user-attachments/assets/b4bb976c-5c9d-4673-96e2-7ee197bf085b